### ISSUE TITLE
Question about batch_size and optimization schedule to reproduce the results in DAIR-V2X-I.

### ISSUE THREAD

**Issue by HaoZhang16613** [ADLab-AutoDrive/BEVHeight#49]
Dear authors, thanks for sharing the great work.
I train BEVHeight model with backbone ResNet101 in DAIR-V2X-I dataset and evaluate the model by your evaluation code, the results in much lower than that reported in the paper. 
The results are as following: (the last row is my reproduce)
<img width="727" alt="image" src="https://github.com/ADLab-AutoDrive/BEVHeight/assets/130417666/3a10d1ee-56d2-432a-bc19-c3cd59b8a5e8">
The training script is: [bev_height_lss_r101_864_1536_256x256_102.py](https://github.com/ADLab-AutoDrive/BEVHeight/blob/main/exps/dair-v2x/bev_height_lss_r101_864_1536_256x256_102.py).

Besides, I'm confused about `scheduler = MultiStepLR(optimizer, [19, 23])` in configure_optimizers(), should we reduce the learning rate at 19 and 23 epochs when training 100 Epochs?
I want to know what batch_size and optimization schedule are needed to reproduce the results.

**Comment by HaoZhang16613** [comment]
The problem has been solved by modifying batch_size to **16**. Although the detection loss was unstable, it worked well. Thanks for sharing the great work!

> Dear authors, thanks for sharing the great work. I train BEVHeight model with backbone ResNet101 in DAIR-V2X-I dataset and evaluate the model by your evaluation code, the results in much lower than that reported in the paper. The results are as following: (the last row is my reproduce) <img alt="image" width="727" src="https://private-user-images.githubusercontent.com/130417666/320171786-3a10d1ee-56d2-432a-bc19-c3cd59b8a5e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI0NTQ1NTAsIm5iZiI6MTcxMjQ1NDI1MCwicGF0aCI6Ii8xMzA0MTc2NjYvMzIwMTcxNzg2LTNhMTBkMWVlLTU2ZDItNDMyYS1iYzE5LWMzY2Q1OWI4YTVlOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwN1QwMTQ0MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NjY3OGM1NDAyMWIwYzEwMDA0ZGI4ZmZmMDQ3Zjc3YjFiN2I3YzkzNjY4YWU4ZDg5MDkwZWIyMTk4YjMyNDQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2IRgUPhSsZnyWrtqfAub4sY017p9sxGDfLkfi39sx7Q"> The training script is: [bev_height_lss_r101_864_1536_256x256_102.py](https://github.com/ADLab-AutoDrive/BEVHeight/blob/main/exps/dair-v2x/bev_height_lss_r101_864_1536_256x256_102.py).
> 
> Besides, I'm confused about `scheduler = MultiStepLR(optimizer, [19, 23])` in configure_optimizers(), should we reduce the learning rate at 19 and 23 epochs when training 100 Epochs? I want to know what batch_size and optimization schedule are needed to reproduce the results.

**Comment by Douglassssss** [comment]
hi, i also meet this problem. I only have 1 RTX 3090 GPU. I trained 150 epoches, but the result is: 
car:3d   AP:66.6437, 55.0688, 55.1606
Pedestrian:3d   AP:25.9357, 24.3166, 24.6220
cyclist:3d   AP:39.0454, 37.9792, 38.2092
Did you only change the batch_size to 16? And what's your experiment setting? Could you show how you solved this problem detailedly?Thank you!!!
